include/boost/date_time/gregorian/greg_month.hpp

```diff
@@ -48,7 +48,7 @@ namespace gregorian {
 
 
   //! Wrapper class to represent months in gregorian based calendar
-  class BOOST_DATE_TIME_DECL greg_month : public greg_month_rep {
+  class BOOST_SYMBOL_VISIBLE greg_month : public greg_month_rep {
```

Comment:
Isn't the other option to just remove the macro at this point?  I see that this is what we have in ptime now so I'm fine with it

Reply:
Yes, I see no reason to keep the classes visible. But you have all similar classes marked visible, so I figured you had a reason and I made these two classes marked as well for consistency. If there is no reason, they can all be un-marked by a separate PR. I can make one, if you want me to.

Reply:
We can do it post 1.73.  There will be another round of enhancements for 1.74 so I'll just clean it up then.  And now if the CI would get around to running...